src/renderer/lib/hooks/useAppListeners.ts

```diff
@@ -229,7 +229,28 @@ export const useAppListeners = () => {
 
   const setDolphinOpen = useDolphinStore((store) => store.setDolphinOpen);
   ipc_dolphinClosedEvent.renderer!.useEvent(
-    async ({ dolphinType }) => {
+    async ({ dolphinType, exitCode }) => {
+      if (exitCode === 0xc0000135) {
+        addToast(
+          `Necessary DLLs for launching Dolphin are missing. Head to the FAQ in the settings to find out how to install them.`,
+          {
+            id: `dolphin-error-0x${exitCode.toString(16)}`,
+            appearance: "error",
+            autoDismiss: false,
+          },
+        );
+      } else if (exitCode) {
+        addToast(
+          `Dolphin encountered an error with code: 0x${exitCode.toString(16)}.
+          If you are having issues, please screenshot this and post it in a support channel in the
+          Slippi Discord.`,
```

Comment:
```suggestion
          `Dolphin exited with error code: 0x${exitCode.toString(16)}.
          Please screenshot this and post it in a support channel in the Slippi Discord for assistance.`,
```

Reply:
I included the first part in the case where people see the error when Dolphin exits poorly when being closed (maybe threading stuff). But I don't mind dropping it if you don't mind more people going to the support channels.

Reply:
I think I would prefer any weird exit codes to be reported. I may add an exception for 3 on windows since that is known to happen when they select `Update`.

Reply:
Yeah I think if it's something they shouldn't need to worry about, we shouldn't show the toast in the first place.